### PR TITLE
ci add license year check

### PR DESCRIPTION
**Pull request by Avvessalom**

### CODE REVIEW

File: .github/workflows/license-year-check.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    - cron: "0 3 1 1 *"
+
+jobs:
+  update-license-year:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions:checkout@v2
+        with:
+          fetch-dept: 0
+      - uses: FantasticFiasco/action-update-license-year@v2
```
Comment (SerVB) [inline .github/workflows/license-year-check.yml:14]: Are you sure that it supports our license? From the [description](https://github.com/FantasticFiasco/action-update-license-year) it seems like it doesn't:

image

File: .github/workflows/license-year-check.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          token: ${{secrets.GITHUB_TOKEN}}
+          path: ./**/*.kt
+      - uses: FantasticFiasco/action-update-license-year@v2
+        with:
+          token: ${{secrets.GITHUB_TOKEN}}
+          path: ./**/*.kts
+      - uses: FantasticFiasco/action-update-license-year@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: ./*.kts
```
Comment (SerVB) [inline .github/workflows/license-year-check.yml:28]: Why do we need these extra steps, doesn't the first one (without parameters) cover everything?..

Comment (Avvessalom) [reply .github/workflows/license-year-check.yml:28]: the firs one without parameters cover only `LICENSE` file

File: .github/workflows/license-year-check.yml (hunk 1 above)
Comment (Avvessalom) [reply .github/workflows/license-year-check.yml:14]: oh, I'am sorry, I forgot this. It works for `client`, but for server I should rewrite this for our license

File: .github/workflows/license-year-check.yml (hunk 2 above)
Comment (SerVB) [reply .github/workflows/license-year-check.yml:28]: Thanks, I see now:

```
    # A path or wildcard pattern specifying files to transform. Multiple paths can be
    # specified using literal styled YAML.
    #
    # Required: false
    # Default: LICENSE
    path: ''
```

Can we then use that "literal styled YAML" to have everything in one step? Here is an example:

https://github.com/FantasticFiasco/action-update-license-year#i-want-to-update-all-my-licenses-i-have-more-than-one
```
steps:
- uses: actions/checkout@v2
  with:
    fetch-depth: 0
- uses: FantasticFiasco/action-update-license-year@v2
  with:
    token: ${{ secrets.GITHUB_TOKEN }}
    path: |
      LICENSE-OPEN-SOURCE
      LICENSE-COMMERCIAL
```

Comment (Avvessalom) [reply .github/workflows/license-year-check.yml:28]: Ty. Tomorrow I'll try it in my test repository

File: .github/workflows/license-year-check.yml (hunk 1 above)
Comment (Avvessalom) [reply .github/workflows/license-year-check.yml:14]: it works for our license, [proof](https://github.com/Avvessalom/test-project/pull/13/files)

**Review by SerVB — CHANGES_REQUESTED** [review]
Still there are multiple steps (and multiple commits in your example repo). Let's merge into one step.

The first step just does nothing since our repo doesn't have `Default: LICENSE` file, so we can safely remove the first step.

**Review by SerVB — APPROVED** [review]

**Pull request merged**